imagetest/testworkflow.go

```diff
@@ -0,0 +1,245 @@
+package imagetest
+
+import (
+	"context"
+	"encoding/xml"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"sync"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
+)
+
+var (
+	client *storage.Client
+)
+
+const (
+	testBinariesPath  = "/out"
+	testWrapperPath   = testBinariesPath + "/wrapper"
+	baseGCSPath       = "gs://gcp-guest-test-outputs/cloud_image_tests/"
+	createVMsStepName = "create-vms"
+)
+
+// TestWorkflow defines a test workflow which creates at least one test VM.
+type TestWorkflow struct {
+	Image          string
+	Name           string
+	ShortImage     string
```

Comment:
What is the difference between Image and ShortImage?

Reply:
image: projects/projectname/global/images/family/debian-10
shortimage: debian-10

i will add a comment to clarify this